modules/p2/ding_serendipity_lists/ding_serendipity_lists.module

```diff
@@ -356,8 +361,10 @@ function ding_serendipity_lists_author_from_lists_serendipity_add($context) {
       return array();
     }
 
+    $field_filters = [];
     // Pick the first author.
     $terms[] = 'facet.creator="' . $author[0] . '"';
+    $field_filters[] = new TingSearchFieldFilter(TingSearchCommonFields::AUTHOR, $author[0], TingSearchFieldFilter::OP_AND);
```

Comment:
Do we need `$terms` above now that we have `$field_filters`?

Reply:
Nope, leftover yet again.